portfolio/src/main/webapp/comments.html

```diff
@@ -27,13 +27,18 @@
     </header>
     
     <!-- Comments -->
+    <!-- Number of comments dropdown -->
+    <label for="comment-max-no">Number of comments to diplay</label>
```

Comment:
As suggested in DataServlet.java remove the "no" from the names. Also, make this `max-comments` to keep the naming consistent with the naming in DataServlet.java.
```suggestion
    <label for="max-comments">Number of comments to diplay</label>
```